packages/commonwealth/server/api/user.ts

```diff
@@ -10,4 +10,7 @@ export const trpcRouter = trpc.router({
   getUserProfile: trpc.query(User.GetUserProfile, trpc.Tag.User),
   getUserAddresses: trpc.query(User.GetUserAddresses, trpc.Tag.User),
   searchUserProfiles: trpc.query(User.SearchUserProfiles, trpc.Tag.User),
+  createReferralLink: trpc.command(User.CreateReferralLink, trpc.Tag.User),
+  getReferralLink: trpc.query(User.GetReferralLink, trpc.Tag.User),
+  getUserReferrals: trpc.query(User.GetUserReferrals, trpc.Tag.User),
```

Comment:
what's the difference between those two? Maybe we should be more specific in naming?

Reply:
The first one returns the link, the second the projected events with referrals